environment/src/main/java/com/sequenceiq/environment/environment/flow/stop/event/EnvStopEvent.java

```diff
@@ -0,0 +1,64 @@
+package com.sequenceiq.environment.environment.flow.stop.event;
+
+import com.sequenceiq.flow.reactor.api.event.BaseNamedFlowEvent;
+
+import reactor.rx.Promise;
+
+public class EnvStopEvent extends BaseNamedFlowEvent {
+
+    public EnvStopEvent(String selector, Long resourceId, String resourceName, String resourceCrn) {
+        super(selector, resourceId, resourceName, resourceCrn);
+    }
+
+    public EnvStopEvent(String selector, Long resourceId, Promise<Boolean> accepted, String resourceName, String resourceCrn) {
+        super(selector, resourceId, accepted, resourceName, resourceCrn);
+    }
+
+    public static final class EnvStopEventBuilder {
+        private String resourceName;
+
+        private String resourceCrn;
+
+        private String selector;
+
+        private Long resourceId;
+
+        private Promise<Boolean> accepted;
+
+        private EnvStopEventBuilder() {
+        }
+
+        public static EnvStopEventBuilder anEnvStopEvent() {
```

Comment:
move this to the built class with name builder()

Reply:
this is the convention in this service